srcpkgs/oilshell/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'oilshell'
+pkgname=oilshell
+version=0.6.0
+revision=1
+wrksrc=oil-${version}
+build_style=configure
+configure_args="--prefix=/usr $(vopt_with readline)"
+makedepends="$(vopt_if readline readline-devel)"
+short_desc="Oil, a new unix shell"
+maintainer="Caleb JA <skywise@tutanota.de>"
+license="Apache-2.0"
+homepage="https://www.oilshell.org"
+distfiles="${homepage}/download/oil-${version}.tar.xz"
+checksum=141e3a5c1997a696d65747966978aaa38921d77e303aad9e77b4ab4aedab84b7
+register_shell="/usr/bin/osh"
+nocross=yes
+nostrip=yes
+
+build_options="readline"
+desc_option_readline="Enables interactive features such as tabbed
```

Comment:
Not needed `common/options.description` already provides a desc for readline.

Reply:
Also for later reference, you never need to do them that detailed.

Reply:
I removed the description.